src/sentry/search/utils.py

```diff
@@ -139,6 +141,8 @@ def parse_datetime_value(value):
     # timezones are not supported and are assumed UTC
     if value[-1:] == "Z":
         value = value[:-1]
+    if len(value) >= 6 and value[-6] == "+":
+        value = value[:-6]
```

Comment:
Should we consider using multiple timeformats like we do on line 159 or is strptime not able to handle the timezone information.

Reply:
Hmmm I assumed based on the comment that we were stripping the timezone. On closer inspection that's not what happens.